### ISSUE TITLE
Batched input of token IDs without attention mask fails

### ISSUE THREAD

**Issue by AdamBelfki3** [ndif-team/nnsight#614]
- `nnsight==0.6.0.a1`
- `torch==2.10.0`
- `transformers==5.2.0`

```py
from nnsight import LanguageModel

model = LanguageModel("openai-community/gpt2")

input_tokens = model.tokenizer("The Eiffel Tower is in the city of", return_tensors="pt")["input_ids"]

with model.trace() as tracer:
    with tracer.invoke(input_tokens) as invoker:
        out = model.output.save()
    
    with tracer.invoke(input_tokens) as invoker2:
        out2 = model.output.save()
```

```
File "nnsight/src/nnsight/intervention/tracing/tracer.py", line 327, in __exit__
    result = super().__exit__(exc_type, exc_value, traceback)
  File "nnsight/src/nnsight/intervention/tracing/base.py", line 689, in __exit__
    self.backend(self)
  File "nnsight/src/nnsight/intervention/backends/execution.py", line 24, in __call__
    raise wrap_exception(e, tracer.info) from None
  File "nnsight/src/nnsight/intervention/backends/execution.py", line 21, in __call__
    return tracer.execute(fn)
  File "nnsight/src/nnsight/intervention/tracing/invoker.py", line 75, in execute
    inputs, batch_group = self.tracer.batcher.batch(
  File "nnsight/src/nnsight/intervention/batching.py", line 180, in batch
    self.batched_args, self.batched_kwargs = batchable._batch(
  File "nnsight/src/nnsight/modeling/language.py", line 330, in _batch
    attention_mask = batched_inputs["attention_mask"]
```
